src/main/java/dude/task/ListManager.java

```diff
@@ -0,0 +1,104 @@
+package dude.task;
+
+import dude.commands.Interface;
+import dude.commands.Parser;
+import exception.*;
+
+import java.util.ArrayList;
+
+public class ListManager {
+    private static final ArrayList<Task> list = new ArrayList<>();
+    private static int index = 0;
+
+    public static Task getTask(int index){
+        return list.get(index) ;
+    }
+    public static ArrayList<Task> getList(){
+        return list;
+    }
+    public static void printList() {
+        System.out.println(Interface.LINE);
+        if(list.size() == 0){
+            System.out.println("List currently empty! Please input tasks");
+            System.out.println(Interface.LINE);
+            return;
+        }
+        for (int i = 0; i < list.size(); i++) {
+            System.out.println((i + 1) + "." + list.get(i));
+        }
+        System.out.println(Interface.LINE);
+    }
+
+    public static Task getString(int index) {
+        return list.get(index);
+    }
+
+    public static void markDone(String index) throws InvalidMarkException {
+        try {
+            int intIndex = Integer.parseInt(index) - 1;
+            if (intIndex >= list.size() | intIndex < 0) {
+                throw new InvalidMarkException();
+            }
+            list.get(intIndex).setDone();
+            Interface.markDoneMessage();
+            System.out.println(list.get(intIndex));
+            System.out.println(Interface.LINE);
+        } catch (NumberFormatException e){
+            throw new InvalidMarkException();
+        }
+    }
+
+    public static void markUndone(String index) throws InvalidUnmarkException {
+        try {
+            int intIndex = Integer.parseInt(index) - 1;
+            if (intIndex >= list.size() | intIndex < 0) {
+                throw new InvalidUnmarkException();
+            }
+            list.get(intIndex).setUndone();
+            Interface.markUndoneMessage();
+            System.out.println(list.get(intIndex));
+            System.out.println(Interface.LINE);

```

Comment:
Instead of using two "System.out.println()" statements, you can just use it once. Example:
System.out.println( list.get(intIndex) + "\n"
            + Interface.LINE );